docs/development/brim.md

```diff
@@ -0,0 +1,14 @@
+# Brim
+
+Brim is the name given to a [technique](https://github.com/gajus/brim) widely 
+adopted by the web industry to workaround issues with Safari's UI 
+intruding into the screen during game play. It allows the user to swipe up, 
+tap the screen or rotate the device to dismiss the browser UI and return the 
```

Comment:
tapping the screen no longer works and rotating to portrait and landscape only works if the game is the only tab open